gradle/model-compiler.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/*
+ * This script configures Spine defaults for the Model Compiler code generation.
+ *
+ * Applies the following interfaces generation configurations:
+ * — all messages in Protobuf files ending with "commands.proto" are marked as `CommandMessage`;
+ * — all messages in Protobuf files ending with "events.proto" are marked as `EventMessage`;
+ * — all messages in Protobuf files ending with "rejections.proto" are marked as `RejectionMessage`;
+ * — all messages that qualify to be UUID messages are marked as `UuidValue`.
+ *
+ * And the following method generations are applied:
+ * — all messages that qualify to be UUID messages have helper `generate` and `of` methods
+ *   generated using `UuidMethodFactory`;
+ * — all messages that have validating builders have helper `vBuilder` and instance `toVBuilder`
+ *   methods generated using `VBuilderMethodFactory`.
+ *
+ * Be aware that the root project `buildscript` should already have a classpath
+ * `io.spine.tools:spine-model-compiler:<version>` classpath dependency.
+ */
+
+modelCompiler {
+    generateInterfaces {
+        mark filePattern().endsWith("commands.proto"), "io.spine.base.CommandMessage"
```

Comment:
Let's have `mark` as a closure. So that it goes:

```groovy
mark {
  files named().withSuffix("commands.proto"), "io.spine.base.CommandMessage"
}
```